src/content/datasheets/core-datasheet.md

```diff
@@ -313,22 +313,27 @@ Mechanical drawings of the Core are available [here.](https://github.com/spark/c
 <table border = '1'>
    <tr>
       <th>Parameter</th>
-      <th>Value</th>
+      <th>Value (imperial)</th>
+      <th>Value (metric)</th>
    </tr>
    <tr>
       <td>Length</td>
       <td>1.47"</td>
+      <td>37.27 mm</td>
```

Comment:
I'm thinking that we might as well not have the PR pulled in if the change is not made since the imperial and metric does not match up to each other...